Resources/Prototypes/_LostParadise/Maps/submarine.yml

```
@@ -1,81 +1,81 @@
-            MailCarrier: [ 1, 2 ]
-            SalvageSpecialist: [ 3, 5 ]
-            Quartermaster: [ 1, 1 ]
-          #justice
-            Prosecutor: [ 1, 1 ]
-            Clerk: [ 1, 1 ]
-            ChiefJustice: [ 1, 1 ]
-          #LOST PARADISE
-            LPPBlueShieldOfficer: [ 1, 1 ]
-            LPPAdjutant: [ 1, 1 ]
-            LPPSecurityEngineer: [ 1, 1 ]
+﻿- type: gameMap
```

yamllint

[error] wrong new line character: expected \n

(new-lines)